### DISCUSSION TITLE
Multiple rAudios ?

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#817] [General]
What does this new feature announced in today's (20220225) release do please? How is it meant to be used ? 

Thanks

Chris

**Comment by hjfisch** [comment]
![c1](https://user-images.githubusercontent.com/93144100/155765118-8ee13ae0-225a-4376-8942-7916c0011158.jpg)
![c2](https://user-images.githubusercontent.com/93144100/155765205-6fc9cc29-f461-406f-90f1-0053ebd43929.jpg)
If you have more than one rAudio-Player , then you can change the player within the  app.

**Reply by ceejayemm** [reply]
If this is the case then I am not sure this is the best way to do this. Why is it necessary to store IP Numbers in the Functions page.  Surely the IP numbers accessible to the Android App (if this what you mean) should be stored in the App not the base system.

Chris

**Comment by DD4WH** [comment]
I tried to install multiple rAudios in my home WiFi network. Both of them work and I can switch between them for example using the SpotifyApp on my smartphone.

However, I cannot switch between them using the rAudio Android App.
I tried to enable the feature using the switch shown in the post above. But regardless of what I enter in the IP adress and name field, the switch always goes back into the disabled position.

What am I supposed to enter ?

* I made a network scan with a WiFi netwark scan app and entered exactly the IP adresses and the SSIDs of the rAudios
* however: no success, multiple rAudio feature remains disabled . . .

Would be nice if someone could help me with information on that feature! Thanks in advance!

**Comment by hjfisch** [comment]
Same for me now ,Switch goes back to Disable.

**Comment by rern** [comment]
@DD4WH,
@hjfisch,

Please try:
```sh
curl -L https://github.com/rern/rAudio-1/raw/UPDATE/srv/http/assets/js/features.js -o /srv/http/assets/js/features.js
```

**Reply by hjfisch** [reply]
> @DD4WH, @hjfisch,
> 
> Please try:
> 
> ```shell
> curl -L https://github.com/rern/rAudio-1/raw/UPDATE/srv/http/assets/js/features.js -o /srv/http/assets/js/features.js
> ```

Thank's ,
Did it with both RPI3 and it's working now.

**Comment by DD4WH** [comment]
Perfect! Thanks a lot! 
* I issued the command on both RPi Zero 2W using putty
* after that I used the Android app to enable the "multiple rAudio" feature --> worked !
* after that, I can now change the rAudio device in the app 

Perfect! Thank you very much for rAudio and your super quick bug fix!
